PascalTriangle.cpp

```diff
@@ -0,0 +1,43 @@
+// Time Complexity :O(n^2) where n in the number of rows 
+// Space Complexity : O(1)  
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : No
+class Solution {
+public:
+    vector<vector<int>> generate(int numRows) {
+        vector<vector<int>> ans;
```

Comment:
try to optimize the redundant code for the cases when a row is 1 and 2. 
Logic is good.